### PR TITLE
IA-3577 Remove test campaigns from calendar download scopes

### PR DESCRIPTION
**Pull request by kemar**
Remove test campaigns from calendar download scopes.

Related JIRA tickets : [IA-3577](https://bluesquare.atlassian.net/browse/IA-3577)

## How to test

See unit tests.

## Print screen

csv

[IA-3577]: https://bluesquare.atlassian.net/browse/IA-3577?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: plugins/polio/api/campaigns/campaigns.py [hunk 1]
```diff
@@ -947,10 +951,7 @@ def create_all_rounds_scopes_csv(self, request, **kwargs):
         org_units_list = []
         # loop on filtered rounds and make the org_units_list to be pushed in the csv file
         for round in rounds:
-            campaign = round.campaign
-            if not campaign:
-                continue
-            org_units_list += self.get_org_units_list(round, campaign)
+            org_units_list += self.get_org_units_list(round, round.campaign)
```
Comment (mathvdh) [inline plugins/polio/api/campaigns/campaigns.py:954]: @kemar Are you sure about this ?

**Review by mathvdh — COMMENTED** [review]
See my comment for the rest it seems good to me

File: plugins/polio/api/campaigns/campaigns.py (hunk 1 above)
Comment (kemar) [reply plugins/polio/api/campaigns/campaigns.py:954]: @mathvdh yup, this is now handled at line 931 with `exclude(Q(campaign__isnull=True)` and covered in tests.

**Review by mathvdh — APPROVED** [review]
👍🏼

**Pull request merged**